R/twidlr-gamlss.R

```diff
@@ -0,0 +1,42 @@
+
+
+
+#' data.frame-first formula-second method for \code{\link[gamlss]{gamlss}}
+#'
+#' This function passes a data.frame, formula, and additional arguments to
+#' \code{\link[gamlss]{gamlss}}.
+#'
+#' @seealso \code{\link[gamlss]{gamlss}}
+#'
+#' @inheritParams twidlr_defaults
+#' @export
+#'
+#' @examples
+#' fit <- gamlss(mtcars, vs ~ hp + wt, family = BI())
+#' summary(fit)
+#' predict(fit, mtcars[1:5,])
+#'
+#' # Help page for function being twiddled
+#' ?gamlss::gamlss
+gamlss <- function(data, formula, ...) {
+  check_pkg("gamlss")
```

Comment:
Please add gamlss package to the "Suggests" field in the [DESCRIPTION file](https://github.com/drsimonj/twidlr/blob/master/DESCRIPTION).